nullaway/src/main/java/com/uber/nullaway/handlers/Handlers.java

```diff
@@ -46,7 +46,7 @@ public static Handler buildDefault(Config config) {
     ImmutableList.Builder<Handler> handlerListBuilder = ImmutableList.builder();
     MethodNameUtil methodNameUtil = new MethodNameUtil();
 
-    if (config.acknowledgeRestrictiveAnnotations()) {
+    if (config.acknowledgeRestrictiveAnnotations() || config.isJSpecifyMode()) {
```

Comment:
Have we considered using: 
`acknowledgeRestrictiveAnnotation = acknowledgeRestrictiveAnnotation OR JSpecifyMode `
at the top level instead of modifying this if condition?

I was asking because if in the future we write more if conditions involving `acknowledgeRestrictiveAnnotation`, we will have to remember to add this `OR JSpecifyMode`  statement in all those places. If we feel like we are unlikely to add any more if conditions around `acknowledgeRestrictiveAnnotation`, we can ignore this comment.